packages/extension-api/src/extension-api.d.ts

```diff
@@ -1494,7 +1491,7 @@ declare module '@podman-desktop/api' {
 
     export function withProgress<R>(
       options: ProgressOptions,
-      task: (progress: Progress<{ message?: string; increment?: number }>, token: CancellationToken) => Promise<R>,
+      task: (progress: Progress<{ message?: string; increment?: number }>, token?: CancellationToken) => Promise<R>,
```

Comment:
token is mandatory

Reply:
a token is to know if the task has been canceled or not, if cancellable is false, it'll just always return that task was never canceled